scripts/test_env.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Good catch!

Reply:
Mac's system bash is old version, that's how I spotted this.